src/ledgerwriter/src/test/java/anthos/samples/bankofanthos/ledgerwriter/LedgerWriterControllerTest.java

```diff
@@ -73,7 +80,26 @@
     @BeforeEach
     void setUp() {
         initMocks(this);
+        StackdriverMeterRegistry meterRegistry = new StackdriverMeterRegistry(new StackdriverConfig() {
```

Comment:
will this need to be added for balancereader as well?

Reply:
Yes! Just did!